internal/view/app.go

```diff
@@ -173,6 +172,49 @@ func (a *App) suggestCommand() model.SuggestionFunc {
 	}
 }
 
+func (a *App) suggestSubCommand(command string) []string {
+	cmds := strings.Split(command, " ")
+	if len(cmds[0]) == 0 || len(cmds) != 2 {
+		return nil
+	}
+
+	var suggests []string
+	switch strings.ToLower(cmds[0]) {
+	case "cow", "q", "q!", "qa", "Q", "quit", "?", "h", "help", "a", "alias", "x", "xray", "dir":
```

Comment:
Don't think this is necessary to check for those since we only care about context or something with this shape aka `xxx ns` ie len = 2 or am I missing it?
I am also thinking we should intro a command type, so we can hang convenience like IsContext() or IsTuple()?? These could be reused in several places and also could open up listing multiple nss i.e `dp ns1 ns2` at some point in the future...

Reply:
> Don't think this is necessary to check for those since we only care about context or something with this shape aka xxx ns ie len = 2 or am I missing it?

There are two reasons for this, the first being that some of the commands in here have a second argument that is not ns, such as `xray dp`. The second is that for e.g. `q` these commands don't really need any more arguments, would it be weird if the user accidentally typed `q x` to see suggestions?

> I am also thinking we should intro a command type, so we can hang convenience like IsContext() or IsTuple()?? These could be reused in several places and also could open up listing multiple nss i.e dp ns1 ns2 at some point in the future...

Yes, it also helps to manage these commands better. It may be a bit tricky to manage with resource commands?